spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -867,18 +867,17 @@ private List<DeleteFile> deleteFiles(Table table) {
   }
 
   private <T extends ContentFile<?>> List<T> except(List<T> first, List<T> second) {
-    Set<String> secondPaths =
-        second.stream().map(f -> f.path().toString()).collect(Collectors.toSet());
+    Set<String> secondPaths = second.stream().map(f -> f.location()).collect(Collectors.toSet());
     return first.stream()
-        .filter(f -> !secondPaths.contains(f.path().toString()))
+        .filter(f -> !secondPaths.contains(f.location()))
         .collect(Collectors.toList());
   }
 
   private void assertNotContains(List<DeleteFile> original, List<DeleteFile> rewritten) {
     Set<String> originalPaths =
-        original.stream().map(f -> f.path().toString()).collect(Collectors.toSet());
+        original.stream().map(f -> f.location()).collect(Collectors.toSet());
```

Comment:
nit: there are a bunch of places like this which can be updated to `.map(ContentFile::location)`